apps/src/code-studio/assets/DefaultSpritesEditor.jsx

```diff
@@ -30,11 +31,23 @@ export default class DefaultSpritesEditor extends React.Component {
       });
   }
 
+  incrementPendingChanges() {
```

Comment:
this function isn't bound, which could cause some weirdness with `this` -- we can either make this an arrow function (which is automatically bound), or manually bind the method in the constructor